desktop/logger.ts

```diff
@@ -1,14 +1,23 @@
 import path from 'path';
 import { app } from 'electron';
+import { captureEvent } from '@sentry/electron';
+import { captureMessage } from '@sentry/react';
 
 const logger = require('electron-log');
 
 logger.transports.file.file = path.join(app.getPath('userData'), '/app-logs.txt');
 logger.transports.console.level = false;
 
 const Logger = ({ className }: { className: string }) => ({
-  log: (fn: string, res: any, args?: any) => logger.info(`${className}, in ${fn}, output: ${JSON.stringify(res)} ${args ? `; args: ${JSON.stringify(args)}` : ''}`),
-  error: (fn: string, err: any, args?: any) => logger.error(`${className}, in ${fn}, error: ${JSON.stringify(err)} ${args ? `; args: ${JSON.stringify(args)}` : ''}`),
+  log: (fn: string, res: any, args?: any) => {
+    logger.info(`${className}, in ${fn}, output: ${JSON.stringify(res)} ${args ? `; args: ${JSON.stringify(args)}` : ''}`);
```

Comment:
Seems like these templates strings should be extracted into function. Then there will be:
```
const msg = formatLogMessage(className, fn, res, args);
logger.info(msg);
captureMessage(msg);
```
And the pretty same thing for error case :)